python/docs/jsonschema_tutorial.rst

```diff
@@ -0,0 +1,180 @@
+*************************************************
+Tutorial: Converting CSV to MCAP with JSON Schema
+*************************************************
+
+Introduction
+------------
+
+In this tutorial we'll take a publicly available dataset and demonstrate how to convert CSV
+data to MCAP. We'll use Python and `JSON Schema <https://json-schema.org/>`_ to get up and running
+quickly. All of the code from this tutorial is runnable, and can be found
+`in the MCAP repo <https://github.com/foxglove/mcap/tree/main/python/examples/jsonschema/pointcloud_csv_to_mcap.py>`_.
+
+Decoding the Data Source
+------------------------
+
+This tutorial uses the public CSV dataset "Sydney Urban Objects Dataset",
+released by the **Australian Centre for Field Robotics** at the University of Sydney, NSW, Australia.
+The original dataset can be downloaded from
+`their website <https://www.acfr.usyd.edu.au/papers/SydneyUrbanObjectsDataset.shtml>`_. This is a
+collection of CSV files containing point clouds collected from objects on the street.
```

Comment:
"The dataset contains point cloud data collected on the road [by autonomous cars?]."